handlers/handlers.go

```diff
@@ -269,6 +269,15 @@ func readDataAggregation(w http.ResponseWriter, req *http.Request, cfg *config.C
 		}
 	}
 
+	if _, rssParam := urlQuery["rss"]; rssParam {
+		req.Header.Set("Accept", "application/rss+xml")
+		if err = createRSSFeed(ctx, w, req, collectionID, accessToken, searchC, validatedQueryParams, template); err != nil {
+			setStatusCode(w, req, err)
```

Comment:
If we're setting an error we should log why ie.
`log.Error(ctx, "failed to create rss feed", err)`